### DISCUSSION TITLE
Database size usage exceeded for new project

### DISCUSSION THREAD

**Discussion by nangemo** [supabase/supabase#21553] [Questions] [answered]
Hey community, I recently created a project using Supabase. The website hasn't launched yet i.e. there is no data feeding into the database yet. I also check usage via the following query:
select
  sum(pg_database_size(pg_database.datname)) / (1024 * 1024) as db_size_mb
from pg_database;
, which showed I used up 80 MB

However, the dashboard says I have exceed the 500 MB limit for the free service. 

Could somebody help with the above?

Is that because Supabase doesn't like a new customer who is not yet ready to pay for their service??

If it is not case, how should I check where is 500mb usage occupied at the moment?

I have tried vacuum etc (regardless the fact that it is not applicable to my case)
Thank you.

**Comment by GaryAustin1** [comment, marked as answer]
I would hard refresh browser and restart instance.  If that does not clear it then contact support.  Something seems wrong with your instance or your dashboard.

80MB is the typical starting size for a new instance with not much user data yet.

**Reply by nangemo** [reply]
Thanks, when I checked again today, the dashboard is now showing the actual usage correctly and I don't have the exceeding the limit restriction anymore. Also, you could be right, a hard refresh could help.

**Comment by nangemo** [comment]
1)wait for another day [verified]
2)hard refresh [not sure, but it could work]
